presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kerberos-kms-hdfs-impersonation/hive.properties

```diff
@@ -9,10 +9,9 @@ hive.allow-rename-column=true
 hive.metastore.authentication.type=KERBEROS
 hive.metastore.service.principal=hive/_HOST@LABS.TERADATA.COM
 # When using HDFS impersonation we talk to HDFS as session user configured in tempto (which is 'hive')
-# However, TODO we don't have Metastore impersonation yet; when DROP TABLE is issued,
-# Metastore needs to be able to delete the table files on HDFS, otherwise they be silently left behind,
-# potentially causing some further tests to fail. For this reason, `hive.metastore.client.principal` needs
-# to match session user configured in tempto.
+# However, when DROP TABLE is issued, Metastore needs to be able to delete the table files on HDFS,
```

Comment:
unrelated

Reply:
this is a separate commit. do you want me to move this commit to a separate PR?

Reply:
this is just nit really